templates/blank/src/app/(frontend)/globals.css

```diff
@@ -0,0 +1,139 @@
+:root {
```

Comment:
Should this be called `styles.css`? Globals implies it effects more than this one page.